test/load_balancer/test_load_balancer.py

```diff
@@ -14,21 +14,12 @@
 
 """Test the Load Balancer unified spec tests."""
 
-import os
 import sys
 
 sys.path[0:0] = [""]
 
 from test import unittest
-
-from test.unified_format import generate_test_classes
-
-# Location of JSON test specifications.
-TEST_PATH = os.path.join(
-    os.path.dirname(os.path.realpath(__file__)), 'unified')
-
-# Generate unified tests.
-globals().update(generate_test_classes(TEST_PATH, module=__name__))
+from test.test_load_balancer import *
```

Comment:
Why keep this file? Is not feasible to modify the `python setup.py test` invocation to only run the tests in `test_load_balancer.py` on LB-enabled clusters?

Reply:
Because then we'd need to run the tests twice, once for  test_load_balancer.py and once for everything in `/test/load_balancer/`. Ideally we only need to run one shell command to run all the tests.

Reply:
👍🏽